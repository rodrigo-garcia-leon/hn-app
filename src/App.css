main {
  margin: 32px auto;
  width: 40vw;
}
