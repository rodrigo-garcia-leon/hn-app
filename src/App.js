import React from "react";
import ApolloClient from "apollo-boost";
import { ApolloProvider } from "@apollo/react-hooks";
import ListView from "./components/ListView";
import "./App.css";

const client = new ApolloClient({
  uri: "https://hn-api.rodrigogarcia.me/graphql"
});

function App() {
  return (
    <ApolloProvider client={client}>
      <main>
        <ListView></ListView>
      </main>
    </ApolloProvider>
  );
}

export default App;
