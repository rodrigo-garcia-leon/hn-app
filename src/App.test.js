import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  test("ok", async () => {
    const element = render(<App />);

    expect(element.container).toMatchSnapshot();
  });
});
