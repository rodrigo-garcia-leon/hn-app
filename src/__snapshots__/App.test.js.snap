// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ok 1`] = `
<div>
  <main>
    <p>
      Loading...
    </p>
  </main>
</div>
`;
